angular/src/components/callout.component.ts

```diff
@@ -36,23 +36,23 @@ export class CalloutComponent implements OnInit {
         this.title = this.i18nService.t("warning");
       }
       if (this.icon === undefined) {
-        this.icon = "fa-warning";
+        this.icon = "bwi-exclamation-triangle";
```

Comment:
I was hoping we could do something like `BitwardenIcons.ExclamationTriangle`, and be certain it exists due to type safety. Will also make it easier to find future references since it will fail to compile.